src/main/resources/logging.properties

```diff
@@ -0,0 +1,4 @@
+handlers=java.util.logging.ConsoleHandler
```

Comment:
why do we need this now? (not against it - just want to understand the rationale)

Reply:
It's just to make any logging appear as if it comes directly from JBang (with our normal "[jbang]" prefix for every line), which was needed to make the logging output from the new module appear exactly the same as it did before.

Reply:
ok but:
a) what makes it end up in stderr not stdout?
b) doesn't this meddling with jul logging make it harder to embed?

Reply:
a) what do you mean? all JBang messages go to stderr, right? (at least any of the `xxxMsg()` methods, which is what the JUL logging replaces here)

b) also not sure what you mean here. Using JUL logging is pretty normal, isn't it? You could use it just fine even with logging completely disabled. There's no essential output being generated using JUL.